src/index.tsx

```diff
@@ -1134,15 +1184,15 @@ export class App extends React.Component<{}, AppState> {
       parsedElements.length > 0 &&
       parsedElements[0].type // need to implement a better check here...
     ) {
-      clearSelection(elements);
+      elements = clearSelection(elements);
 
       let subCanvasX1 = Infinity;
       let subCanvasX2 = 0;
       let subCanvasY1 = Infinity;
       let subCanvasY2 = 0;
 
-      const minX = Math.min(...parsedElements.map(element => element.x));
-      const minY = Math.min(...parsedElements.map(element => element.y));
+      //const minX = Math.min(parsedElements.map(element => element.x));
```

Comment:
This PR broke the ability of center pasted elements under the cursor

Reply:
Where's that code example from? Can't find any such change from this PR.

Reply:
This is indeed weird change. Didn't catch it.

Reply:
@Fausto95 fix in #291